content/en-us/chat/voice-chat.md

```diff
@@ -37,6 +39,110 @@ If you previously set the maximum number of users in a place to more than 50, yo
 
 When you update the maximum number of users in a place to fewer than 50, there may be servers already configured to a different, higher number. Since those servers won't support voice chat, it's recommended to [restart servers](../production/publishing/publish-experiences-and-places.md#update-experiences).
 
+### Using the Audio API to control Voice Chat
+
+By setting `Class.VoiceChatService.UseAudioApi|UseAudioApi` to `Enabled`, participant voices will be controlled by `Class.AudioDeviceInput|AudioDeviceInputs`.
+
+Scripts can use the properties and methods of `Class.AudioDeviceInput|AudioDeviceInputs` to implement custom, game-specific logic.
+
+For example, if you wanted to implement team-based chat – where only teammates can hear one another – you could do something along the lines of
+
+```lua
+local function findAudioInput(forPlayer : Player) : AudioDeviceInput?
+	-- Assumes that there is only one AudioDeviceInput per player
+	-- and that it is parented to the Player object
+	-- may need to be reworked for generality if your scene does not match this
+	return forPlayer:FindFirstChildWhichIsA("AudioDeviceInput")
+end
```

Comment:
I think most ppl will want a full working example, which might be lengthier than we'd like in this overall page.
For TextChat, we have a full examples subpage where we go into greater detail on the code samples.

For here, I think it would be beneficial to use this sample to onboard users to the AudioApi in general. IIRC, users would need to supply their own AudioDeviceInput for this to work.